util/OptionsDB.h

```diff
@@ -202,10 +221,20 @@ class FO_COMMON_API OptionsDB
     void        AddFlag(const std::string& name, const std::string& description,
                         bool storable = true)
     {
-        if (m_options.find(name) != m_options.end())
-            throw std::runtime_error("OptionsDB::AddFlag<>() : Option " + name + " was specified twice.");
-        m_options[name] = Option(static_cast<char>(0), name, false, boost::lexical_cast<std::string>(false),
-                                 description, 0, storable, true);
+        std::map<std::string, Option>::iterator it = m_options.find(name);
+        bool value = false;
+        if (it != m_options.end()) {
```

Comment:
Would be helpful to add a comment before this if to explain what that it's checking for any unregistered stored value to use as the set value of the option being added.